foolbox/plot.py

```diff
@@ -57,7 +59,7 @@ def images(
         nrows=nrows,
         figsize=figsize,
         squeeze=False,
-        constrained_layout=True,
+        constrained_layout=False,
```

Comment:
Changing this to false not only prevents weird bounding with labels, but is also just makes the scale parameter more respected.